packages/shared/lib/core/network/tests/network.test.ts

```diff
@@ -40,125 +44,91 @@ describe('File: network.ts', () => {
-        iota: {
-            [NetworkType.Mainnet]: {
-                id: 'iota-mainnet',
-                name: 'IOTA',
-                protocol: NetworkProtocol.IOTA,
-                type: NetworkType.Mainnet,
+    const EXPECTED_NETWORK: Readonly<{ [key in NetworkId]?: INetwork }> = {
+        [NetworkId.Iota]: {
+            id: NetworkId.Iota,
+            name: 'IOTA',
+            protocol: {
+                version: 1,
+                networkName: 'iota',
                 bech32Hrp: 'iota',
-                baseToken: BASE_TOKEN[NetworkProtocol.IOTA],
-                rentStructure: {
-                    vByteCost: 500,
-                    vByteFactorData: 10,
-                    vByteFactorKey: 1,
-                },
-            },
-            [NetworkType.Devnet]: {
-                id: 'iota-devnet',
-                name: 'IOTA Devnet',
-                protocol: NetworkProtocol.IOTA,
-                type: NetworkType.Devnet,
-                bech32Hrp: 'atoi',
-                baseToken: BASE_TOKEN[NetworkProtocol.IOTA],
+                minPowScore: 1500,
+                belowMaxDepth: 15,
                 rentStructure: {
                     vByteCost: 500,
-                    vByteFactorData: 10,
-                    vByteFactorKey: 1,
+                    vByteFactorData: 1,
+                    vByteFactorKey: 10,
                 },
+                tokenSupply: TokenSupply.Iota,
             },
-            [NetworkType.PrivateNet]: <INetwork>{
-                name: 'Private Net',
-                protocol: NetworkProtocol.IOTA,
-                type: NetworkType.PrivateNet,
-            },
+            baseToken: EXPECTED_BASE_TOKEN[NetworkProtocol.IOTA],
         },
-        shimmer: {
-            [NetworkType.Mainnet]: {
-                id: 'shimmer-mainnet',
-                name: 'Shimmer',
-                protocol: NetworkProtocol.Shimmer,
-                type: NetworkType.Mainnet,
+        [NetworkId.Shimmer]: {
+            id: NetworkId.Shimmer,
+            name: 'Shimmer',
+            protocol: {
+                version: 2,
+                networkName: 'shimmer',
                 bech32Hrp: 'smr',
-                baseToken: BASE_TOKEN[NetworkProtocol.Shimmer],
+                minPowScore: 1500,
+                belowMaxDepth: 15,
                 rentStructure: {
-                    vByteCost: 500,
-                    vByteFactorData: 10,
-                    vByteFactorKey: 1,
+                    vByteCost: 100,
+                    vByteFactorData: 1,
+                    vByteFactorKey: 10,
                 },
+                tokenSupply: TokenSupply.Shimmer,
             },
-            [NetworkType.Devnet]: {
-                id: 'testnet',
-                name: 'Testnet',
-                protocol: NetworkProtocol.Shimmer,
-                type: NetworkType.Devnet,
+            baseToken: EXPECTED_BASE_TOKEN[NetworkProtocol.Shimmer],
+        },
+        [NetworkId.Testnet]: {
```

Comment:
Since `Shimmer` and `Testnet` are likely to be the same, it could reduce some code to create a common constant for `Shimmer` and use that, overriding where necessary for `Testnet`